torchgeo/transforms/intensity.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""TorchGeo intensity transforms."""
+
+import torch
+from kornia.augmentation import IntensityAugmentationBase2D
+from torch import Tensor
+
+
+class ToDecibelScale(IntensityAugmentationBase2D):
+    """Convert input tensor to decibel scale.
```

Comment:
Maybe we should clarify that this is power to decibel, as opposed to amplitude to decibel.